modules/ROOT/pages/7.1.2-release-notes.adoc

```diff
@@ -156,6 +156,13 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 // CCFR here.
 
+=== CSS color values set to `transparent` were incorrectly converted to `+#000000+`.
+// #TINY-10916
+
+Previously, transparent color values were incorrectly converted to `+#00000000+`, deviating from the behavior observed in earlier major versions.  Additionally, {productname} does not export the last two digits of the hex string, so the alpha value is lost, causing transparent to be converted to `+#000000+`.
```

Comment:
```suggestion
Previously, transparent color values were incorrectly converted to `+#000000+`, deviating from the behavior observed in earlier major versions.  Additionally, {productname} does not export the last two digits of the hex string, so the alpha value is lost, causing transparent to be converted to `+#000000+`.
```
Shouldn't this be 6 digits?

Reply:
It's a technicality converted to 8 digits, and dumped the last two, so customers will see 6 digits. On second thought 6 digits might be more accurate for customers.